src/plugins/controls/public/options_list/components/options_list_editor_options.tsx

```diff
@@ -102,8 +130,43 @@ export const OptionsListEditorOptions = ({
               if (initialInput?.existsSelected) onChange({ existsSelected: false });
             },
           }}
+          data-test-subj={'optionsListControl__hideExistsAdditionalSetting'}
         />
       </EuiFormRow>
+      <EuiFormRow>
+        <>
+          <EuiSwitch
+            label={OptionsListStrings.editor.getHideSortingTitle()}
```

Comment:
> **Warning**
> Tag docs when ready for review

I would appreciate some help with the wording for the "Hide sort" setting and with the title of the additional form that shows up when this setting is set to `false` 🙇  Here's a quick GIF of this in action:

![Nov-16-2022 15-33-44](https://user-images.githubusercontent.com/8698078/202543153-44daf4cb-4d76-4bfa-a40e-64d8471f6a3f.gif)

So there are two pieces of copy to consider: (1) `"Allow suggestions to be sorted"` and (2) `"Suggestions sorting"`

I think (1) is already pretty straightforward - if this setting is set to `false`, this hides the sorting menu from the control popover. Perhaps `"Allow suggestions to be dynamically sorted"` might be more robust? But that might be too much 😆 

The wording of (2) is more difficult - basically, if the author decides that they don't want to give users the option to dynamically sort the suggestions in the popover, then they most provide a default sorting method to use for this control - hence, why the secondary form shows up when (1) is toggled to `false`. I'm not sure if `"Default suggestion sort"` or something equivalent would reflect this better? Definitely need some help with this one